src/DotNetInternals/Pages/Razor.razor

```diff
@@ -101,6 +112,10 @@
         await RefreshAsync();
         compiled = RazorCompiler.Compile(text);
         UpdateOutput();
+
+        var slug = Compressor.Compress(text);
```

Comment:
In my app I took the code from, I just join the fields with `(char)7` since it's unlikely someone will type it in, then split the string again when decoding :)